src/dom/index.js

```diff
@@ -52,10 +52,21 @@ export function setAccessor(node, name, old, value, isSvg) {
 		}
 		if (value && typeof value==='object') {
 			if (typeof old!=='string') {
-				for (let i in old) if (!(i in value)) node.style[i] = '';
+				for (let i in old) if (!(i in value)) {
+					if (i.indexOf('--') === 0) {
```

Comment:
I think we can always use `setProperty()` / `removeProperty()` here, which saves a bunch of bytes.  AFAIK `style.setProperty('color', 'red')` is equivalent to `style.color='red'`.

Reply:
Damn GitHub collapsing comments. Missed this